injective_functions/functions_schemas.json

```diff
@@ -778,6 +778,20 @@
         ]
       }
     },
+    {
```

Comment:
Why does this schema exist in both function_schema and functions_schema? Actually why are there two separate schema files?

Reply:
I’ve placed it only inside staking_schema.json.